CHANGELOG_UNRELEASED.md

```diff
@@ -10,7 +10,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
-- in `bigop.v`, added lemmas `sumnB` and `card_bseq`.
+- in `bigop.v`, added lemma `sumnB`.
+- in `bigop.v`, added lemma `card_bseq`.
```

Comment:
@affeldt-aist  why?

Reply:
isn't it the "do not edit former entries" recommendation?